src/video_core/gpu.h

```diff
@@ -228,6 +252,9 @@ class GPU {
     /// Notify rasterizer that any caches of the specified region should be flushed and invalidated
     virtual void FlushAndInvalidateRegion(CacheAddr addr, u64 size) = 0;
 
+protected:
+    virtual void TriggerCpuInterrupt(const u32 syncpoint_id, const u32 value) const = 0;
```

Comment:
```suggestion
    virtual void TriggerCpuInterrupt(u32 syncpoint_id, u32 value) const = 0;
```

This should also have documentation.